### PR TITLE
Add 3 classes concerned Action.

### PR DESCRIPTION
**Pull request by dairyo**
To adapt thing-if AndroidSDK APIs, I added following 3 classes:

* `AliasAction` ([Android version](https://github.com/KiiPlatform/thing-if-AndroidSDK/blob/develop/thingif/src/main/java/com/kii/thingif/command/AliasAction.java))
* `ActionResult` ([Android version](https://github.com/KiiPlatform/thing-if-AndroidSDK/blob/develop/thingif/src/main/java/com/kii/thingif/command/ActionResult.java))
* `AliasActionResult` ([Android version](https://github.com/KiiPlatform/thing-if-AndroidSDK/blob/develop/thingif/src/main/java/com/kii/thingif/command/AliasActionResult.java))

Related PR: #265

### CODE REVIEW

File: ThingIFSDK/ThingIFSDK/ActionResult.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+//  Copyright (c) 2017 Kii. All rights reserved.
+//
+
+import Foundation
+
+open class ActionResult: NSCoding {
+
+    open let succeeded: Bool
+    open let actionName: String
+    open let data: [String : Any]?
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFSDK/ActionResult.swift]: I think data is `Any` type.

**Comment by dairyo** [comment]
Sorry. I need to do more in this PR.

**Comment by dairyo** [comment]
Finished to fix.

**Comment by yongpingchen** [comment]
LGTM

**Pull request merged**